### DISCUSSION TITLE
スクリプトを使ってフォントを変えたいです

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#141] [Question-original]
## (1) 実現したいこと
スクリプトを使ってフォントを変えたいです
## (2) 実現するために自分が試した内容とその結果
フォントをXcodeに入れて、適用するとこまでできました。しかし、スクリプトからはフォントを変えられないです！
override func viewDidLoad()によって起動時から「yamafont」になる想定ですが、通常フォントの小さい文字になってしまう。
## (3) 調べた内容や資料
https://frog9.com/post/20
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
不明です。エディターではフォントは適用できているので、コードの記述方法がどこかよくない？と思っています。
## (5) GitHubのリンク
https://github.com/KOTARO1209/CountTest

**Comment by txxxxc** [comment]
ビルドに含めるファイルの中にフォントのファイルが含まれていないのが原因なので、以下の画像を参考に追加してみてくださいー！
<img width="1136" alt="image" src="https://user-images.githubusercontent.com/22112440/219907969-20c04295-7fe8-4651-8bc8-efd527dd8af4.png">
